src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -342,11 +342,17 @@ class ComponentBuilderImpl<T extends azdata.Component, TPropertyBag extends azda
 	}
 }
 
-class ContainerBuilderImpl<TComponent extends azdata.Component, TLayout, TItemLayout, TPropertyBag extends azdata.ComponentProperties> extends ComponentBuilderImpl<TComponent, TPropertyBag> implements azdata.ContainerBuilder<TComponent, TLayout, TItemLayout, TPropertyBag> {
+class ContainerBuilderImpl<TComponent extends azdata.Component, TLayout, TItemLayout, TPropertyBag extends azdata.ContainerProperties> extends ComponentBuilderImpl<TComponent, TPropertyBag> implements azdata.ContainerBuilder<TComponent, TLayout, TItemLayout, TPropertyBag> {
 	constructor(componentWrapper: ComponentWrapper) {
 		super(componentWrapper);
 	}
 
+	override withProps(properties: TPropertyBag): azdata.ContainerBuilder<TComponent, TLayout, TItemLayout, TPropertyBag> {
```

Comment:
Changed this to call super.withProps too so we don't have to duplicate the logic (and only have to fix in one place if needed). If there's specific logic needed for containers in the future then we can add it here in addition.